api/ccapi/src/factory.cpp

```diff
@@ -138,5 +140,15 @@ createLearningRateScheduler(const std::string &type,
   return ac.createObject<ml::train::LearningRateScheduler>(type, properties);
 }
 
+std::string getVersion() {
+  std::string version = std::to_string(VERSION_MAJOR);
+  version += ".";
+  version += std::to_string(VERSION_MINOR);
+  version += ".";
+  version += std::to_string(VERSION_MICRO);
+
+  return version;
```

Comment:
```suggestion
  return std::to_string(VERSION_MAJOR) + "."  + std::to_string(VERSION_MINOR) + "."  + std::to_string(VERSION_MICRO);
```
not a big deal, but a suggestion.